packages/docs/.storybook/manager-head.html

```diff
@@ -53,3 +53,55 @@
   name="twitter:image"
   content="https://solid-design-system.fe.union-investment.de/docs/images/sds-signet_meta.png"
 />
+
+<style>
+  #storybook-explorer-tree .sidebar-item {
+    color: #00358e;
+    border-left: 4px solid transparent;
+  }
+
+  #storybook-explorer-tree .sidebar-item[data-selected='true'] {
+    background: transparent;
+    border-radius: 0;
+    border-color: #2d9d00;
+  }
+
+  #storybook-explorer-tree .sidebar-item:hover,
+  #storybook-explorer-tree .sidebar-item[data-nodetype='group']:hover,
+  #storybook-explorer-tree .sidebar-item[data-selected='true']:hover {
+    --tree-node-background-hover: #e9e9e9;
+    background: #e9e9e9;
+    color: #00358e;
+  }
+
+  #storybook-explorer-tree .sidebar-item[data-nodetype='component']:not(:hover),
+  #storybook-explorer-tree .sidebar-item[data-nodetype='group']:not(:hover) {
+    background: transparent;
+  }
+
+  #storybook-explorer-tree .sidebar-item a svg {
+    color: #00358e;
+  }
+
+  div[data-testid='tooltip'] button[id^='list-item-']:hover {
+    background: rgba(0, 53, 142, 0.14);
+  }
+
+  .sb-bar button:focus {
+    box-shadow: rgb(0, 53, 142) 0px 0px 0px 1px inset !important;
+  }
+
+  .sb-bar button:hover {
+    color: #00358e !important;
+    background: rgba(0, 53, 142, 0.14) !important;
+  }
+
+  .sb-bar a:focus-visible,
+  .sb-bar button:focus-visible,
+  .sidebar-container a:focus-visible,
+  .sidebar-container button:focus-visible,
+  .sidebar-container .search-field:has(input:focus-visible) {
+    outline: 2px solid #00358e !important;
```

Comment:
The color `#00358e` is being used in multiple places, maybe we could replace it with a variable, what do you think? Would make it easier to update if needed.